### ISSUE TITLE
Play button does not work.

### ISSUE THREAD

**Issue by stuboo** [twilio/OpenVBX#402]
v1.2.20 r79
mac OS Sierra 10.12.5
Chrome 59.0.3071.115 (Official Build) (64-bit)
running on Dreamhost

When I load the inbox and click the 'play' button, nothing happens.

Debug console shows:

```
www.mydomain.com/:568 Uncaught TypeError: Message.Player.currentSoundObject.stop is not a function
    at _togglePlayForMessage (www.mydomain.com/:568)
    at www.mydomain.com/:568
    at Object.getRecordingUrls (www.mydomain.com/:563)
    at Object.togglePlayForMessage (www.mydomain.com/:568)
    at HTMLAnchorElement.<anonymous> (www.mydomain.com/:576)
    at HTMLAnchorElement.handle (www.mydomain.com/:297)
    at HTMLAnchorElement.k (www.mydomain.com/:297)
```

**Comment by aeqquus** [comment]
I'm not a real programmer so this fix may have hazards of which I am not aware but it is working for me.  I went to the soundmanager2 website and downloaded the current source code.  I backed up the old soundmanager scripts installed with OpenVBX on my server and then copied the new js scripts.  The folder is under "Assets/j".   You'll need to have Flash enabled if you are using chrome.  You should have the following files in the folder.  This works and I haven't had any problems so far.  It is probably an issue with the OpenVBX project using an old SoundManager API.  Unforunately OpenVBX is no longer supported but I use it for my small one man office and it works fine and keeps me up to date on calls and messages.

![filelist](https://user-images.githubusercontent.com/9135906/28542671-99039466-7082-11e7-9166-b7d1818f6ef2.png)

**Comment by mdw123** [comment]
@aeqquus +1

Oddly, I've had playback work intermittently, but updating soundmanager2 seems to fix the issue.

**Comment by tripflex** [comment]
@aeqquus this worked for me!! thank you!!!

**Comment by hamilton9** [comment]
This fix works. Thank you

**Comment by gegere** [comment]
Nice job, I have added this to a branch under a repo I am using in production with a few clients, tested and resolves several issues. I have merged in some updates from @jmtorres and @jpwalters 

Review the branch as my HEAD / default repo here:
https://github.com/gegere/OpenVBX/tree/custom/1.2.22

**Comment by mdw123** [comment]
Since Twilio doesn't support openvbx anymore, should we nominate a new
master branch for this project?

**Comment by gegere** [comment]
Yes, this there a list of people will and able to form a org and start to move this system forward? 

I personally think it is still a great system and will continue to grow if a community can form around it.

**Comment by husunu** [comment]
I am rebuilded Openvbx system using latest twilio libraries ,APis and also latest Codeignite verison,
please contact me for updated OpenVBX
mail :- husainnki@gmail.com
SKYPE : muhammedhusaintk
mobile : +919562470582

twilio.js 1.2 will be depreciated today feb 14, 2018

Warning: Twilio Client 1.2 is deprecated and will reach end of life on February 14, 2018. On this date, Twilio will stop serving this file through the CDN and will begin refusing connections used by this library. We recommend updating to 1.4 as soon as possible. See the full documentation at https://www.twilio.com/docs/api/client/twilio-js-14 for more information.

**Comment by AntoineWtrnd** [comment]
You don't need to add or edit nothing.

This only mean that your browser don't have Adobe Flash enabled.

Not sure why they used Flash for that but anyway, OpenVBX is seriously outdated.

**Comment by husunu** [comment]
manualy update the soundmanager2 library , then button will works
